src/components/Form/Number/Number.jsx

```diff
@@ -19,9 +20,10 @@ export default class Number extends ValidationElement {
       return
     }
 
-    const old = this.state.max
-    this.setState({ max: next.max, value: next.value }, () => {
-      if (old !== next.max) {
+    const oldmax = this.state.max
```

Comment:
This is an instance where local state can be removed from a component. I'm not completely convinced that local state is required in this component. However, this is refactor is probably outside the scope of this PR. I just wanted to point out this anti-pattern in the codebase.